server/jetstream_api.go

```diff
@@ -4120,7 +4120,9 @@ func (s *Server) jsConsumerListRequest(sub *subscription, c *client, _ *Account,
 	}
 
 	for _, o := range obs[offset:] {
-		resp.Consumers = append(resp.Consumers, o.info())
+		if cinfo := o.info(); cinfo != nil {
```

Comment:
This means consumer got deleted yes? So we should remove entry?

Reply:
This just stops `nil` entries ending up in the consumer list response yep.